### PR TITLE
GEOMESA-1811 GeoMesa classpath command can hang and prints duplicate …

### PR DESCRIPTION
**Pull request by jnh5y**
…files

* The GeoMesa 'classpath' command can print multiple copies of the same file if
it is referenced multiple times on the various Hadoop classpaths.

Additionally, calculating the classpath in Bash can hang in some situations.

Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa [hunk 1]
```diff
@@ -109,9 +109,7 @@ if  [[ $1 = configure ]]; then
   geomesaConfigure
 elif [[ $1 = classpath ]]; then
   CLASSPATH="$GEOMESA_CP:$ACCUMULO_CP:$HADOOP_CP:${ZOOKEEPER_JAR}"
-  for element in ${CLASSPATH//:/ } ; do
-    echo ${element}
-  done
+  echo $CLASSPATH | tr ':' '\n' | xargs ls -di | sort -u -k1,1 | awk '{print $2}' | sort
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: does the pipe through `ls` handle missing files? i.e. do we still see them after piping through the sort, etc

Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: Ah, no.  Actually, my fix is listing the actual files (which is different maybe...)

Eh, I think I'll close this PR.  The ticket might rot for a bit (unless someone else posts on it).

Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: what is the scenario that causes it to hang?

Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: we could do something like:
`echo $CLASSPATH | tr ':' '\n' | sort | unique`

**Pull request closed without merging**